Pull request tutorial added to Developement policy

koojozsef opened the pull request.

koojozsef committed:
mini pull request tutorial added to Developement policy

koojozsef commented:
Na el is csesztem, mert az egyik feature branch-ről ágaztam le, nem a masterről... 🥇 
Sebaj, nincs conflict

Jugis reviewed (commented):
Hol írja azt hogy a feature branch amit vissza akarsz mergelni honnan lett származtatva?

koojozsef commented:
Insights tab -> Network
Itt látszik hogy nem a master-es commitból van
csak azért lehetne gond, mert nem a legfrissebb verzióból indultam ki

koojozsef commented:
I corrected the branch:
Rebased the branch to the head of 'master'
`$ git checkout feature/pull-request-description`
`$ git rebase master`
`$ git push origin`
